docs-source/content/_index.md

```diff
@@ -24,11 +24,12 @@ using the operator to deploy and run a WebLogic domain container-packaged web ap
 #### Current production release
 
 The [current release of the operator](https://github.com/oracle/weblogic-kubernetes-operator/releases) is 3.0.0.
-This release was published on July 1, 2020. See the operator prerequisites and supported environments [here]({{< relref "/userguide/introduction/introduction#operator-prerequisites" >}}).
+This release was published on July 17, 2020. See the operator prerequisites and supported environments [here]({{< relref "/userguide/introduction/introduction#operator-prerequisites" >}}).
 
-This release introduces _non-backward compatible_ changes; however, this release can be run in the same
-cluster as the 2.6.0 version of the operator allowing for staged migration.  You can also upgrade from 2.6.0 to 3.0.0 without needing to restart or recreate
-any existing domains.
+This release introduces _non-backward compatible_ changes; however, operators using this release can be run in the same
+Kubernetes cluster as operators using the 2.6.0 version allowing for staged migration.  You can replace an operator using 2.6.0 with an operator using a 3.x release without needing to recreate
```

Comment:
Slightly simpler wording if you state, "You can replace a 2.6.0 operator with a 3.x operator without needing to recreate any existing domains"